Third-party licenses/spdlog.txt

```diff
@@ -0,0 +1,26 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Gabi Melman.                                       
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+-- NOTE: Third party dependency used by this software --
+This software depends on the fmt lib (MIT License),
+and users must comply to its license: https://raw.githubusercontent.com/fmtlib/fmt/master/LICENSE

```

Comment:
Why don't we simply do just like spdlog and mention 3rd party licenses at the bottom of ours ?

Reply:
> Why don't we simply do just like spdlog and mention 3rd party licenses at the bottom of ours ?

We can’t simply link to third-party licenses at the bottom of our license because we’re distributing binaries. Each of the third-party licenses says that we must include a copy of the license, not just a link.

Additionally, I prefer keeping the license for each component in a separate file. I prefer it that way because I think that it will make mistakes less likely. When I look at open-source licenses, I typically don’t read the entire text file. Instead, I look at it and think something along the lines of “oh, this is the GPL, I’ve seen that before”. If a project embedded additional terms at the bottom of the GPL, I would probably miss them. Additionally, I think that it will be easier to add or remove dependencies if we keep their licenses in separate files. I just think that’s it’s easier to add or remove an entire file than it is to add or remove a portion of a file.

That being said, I change this pull request to include a copy of each of the third-party license at the bottom of `LICENSE` if you think that it would be better that way.